Src/Sarif.PatternMatcher/MatchExpression.cs

```diff
@@ -21,6 +21,8 @@ public class MatchExpression
 
         public string DeprecatedName { get; set; }
 
+        public string HelpUri { get; set; }
```

Comment:
>HelpUri [](http://example.com/codeflow?start=22&length=7)

This is fine for now. Sometimes people provide a base URL and then a relative rule topic URL. This can reduce space and also allow for multiple versions of docs to be hosted.

BUT DON'T CHANGE YOUR CODE. :) We can make this improvement later, if it looks like a good idea. What you've done is great for now! :)

Reply:
interesting!